rwengine/src/objects/CharacterObject.cpp

```diff
@@ -497,7 +494,7 @@ void CharacterObject::setJumpSpeed(float speed) {
 
 void CharacterObject::resetToAINode() {
     auto nodes = engine->aigraph.nodes;
-    bool vehicleNode = !!getCurrentVehicle();
+    bool vehicleNode = getCurrentVehicle() != nullptr;
```

Comment:
imo such comparisons in assignments should always be in braces. What does our style guide say @danhedron ?